### PR TITLE
Finish task 6

### PR DESCRIPTION
**Pull request by 1986-OldMan**

### CODE REVIEW

File: pages/list.html [hunk 1]
```diff
@@ -4,10 +4,19 @@
...
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../public/css/list.css">
     <title>List</title>
 </head>
 <body>
-    <a href="../index.html">Home</a>
+    
+    <div class="list-section">
+        <div class="list-section_navigation">
+        <a href="../index.html" class="Home">Home</a>
```
Comment (LabiciIvan) [inline pages/list.html]: Change class name from Home to home, no big letter at begging of class

**Review by LabiciIvan — CHANGES_REQUESTED** [review]

File: pages/list.html (hunk 1 above)
Comment (1986-OldMan) [reply pages/list.html]: Ok

File: pages/list.html [hunk 2]
```diff
@@ -4,10 +4,19 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../public/css/list.css">
     <title>List</title>
 </head>
 <body>
-    <a href="../index.html">Home</a>
```
Comment (LabiciIvan) [inline pages/list.html:10]: Missing class name ? 
Please add.

**Review by LabiciIvan — CHANGES_REQUESTED** [review]

File: pages/list.html (hunk 2 above)
Comment (1986-OldMan) [reply pages/list.html:10]: ok

**Review by LabiciIvan — APPROVED** [review]

**Pull request merged**